postgresqleu/invoices/models.py

```diff
@@ -234,11 +234,19 @@ def __str__(self):
     class Meta:
         ordering = ('-id', )
 
+class VatPercentage(models.DecimalField):
```

Comment:
HTML and PDF templates reference ".vatpercent" directly and I don't want to require changes to those (since some of them are outside this repo).  Creating a new type allows me to normalize the decimals when we read them from the database.  Otherwise they output to strings as "5.000000" instead of "5".

I am open to a better way of doing this.